### ISSUE TITLE
Wrong Facebook informations are crashing the whole app

### ISSUE THREAD

**Issue by SPQRInc** [heiseonline/shariff-backend-php#125]
Hello,

I I do something like 

```
array:4 [▼
  "cache" => array:2 [▼
    "ttl" => 0
    "cacheDir" => "/home/web2/html/site/tmp/cache"
  ]
  "domains" => array:1 [▼
    0 => "example.org"
  ]
  "services" => array:7 [▼
    0 => "Facebook"
    1 => "GooglePlus"
    2 => "LinkedIn"
    3 => "Pinterest"
    4 => "Xing"
    5 => "AddThis"
    6 => "Flattr"
  ]
  "Facebook" => array:2 [▼
    "app_id" => "1234567890"
    "secret" => "terces"
  ]
]
```

And let this return this way:

```
$shariff = new Backend( $configuration );
		$counts  = $shariff->get( $ref ); //site's url
		
		return $counts;
```

The return is always `[ ]`

If I exclude `Facebook` the return is:

`{"googleplus":1,"linkedin":0,"pinterest":0,"xing":0,"addthis":0,"flattr":0}` So it seems that the wrong FB App information are crashing the whole thing?

**Comment by liayn** [comment]
Should be fixed with d607c7ccf67e35ffe62d4903b449e71d8048731c

**Comment by compeak** [comment]
Release [7.1.1](https://github.com/heiseonline/shariff-backend-php/releases/tag/7.1.1) contains a fix for this problem.
